.gitignore

```diff
@@ -1,18 +1,19 @@
-# See http://help.github.com/ignore-files/ for more about ignoring files.
-
-# dependencies
-node_modules
-bower_components
-
-# testing
-coverage
-
-# production
-build
-
-# misc
-.DS_Store
-.env
-npm-debug.log
-yarn.lock
-public/polymer-components.vulcanized.html
\ No newline at end of file
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Usually we just keep this to: https://github.com/BrightspaceUI/status-indicator/blob/master/.gitignore

Reply:
That looks like it would do the job if [this variants entry](https://github.com/Brightspace/d2l-image/pull/8/files#diff-0a08a7565aba4405282251491979bb6bR16) wasn't there. The polymer [hybrid elements page](https://www.polymer-project.org/2.0/docs/devguide/hybrid-elements#dependency-variants) says those are needed if `d2l-image` is intended to be used in polymer 1.x projects. Is that something we want to allow? If not I can remove `variants`.

Reply:
We only need the variants if the hybrid component is going to be used with Polymer 1, which I think is a scenario we will encounter for this component...?

With the variants installed, `polymer serve` will launch the demo (something we should have) on two different ports (usually Polymer 2 on `8081` and Polymer 1 on `8001`), and we can demo/unit test the component under both those scenarios.  

Also, `polymer test` will also run against both variants, and we usually set up the tests to run with shadow dom and shady dom.  For example: https://github.com/BrightspaceUI/status-indicator/blob/master/test/index.html

Reply:
Cool, if `variants` is required then `bower_components*` and `bower-*` are required in `.gitignore`. Running `polymer install --variants` creates a `bower_components-1.x` folder and a `bower-1.x.json`. I'll copy over everything else but keep those two.